add a landing page

flynnduism opened the pull request:
This PR adds a simple landing page (via github pages) with some high level info about the project.

# preview

![Screen Shot 2019-05-15 at 1 45 49 PM](https://user-images.githubusercontent.com/686194/57808239-c97ee300-7717-11e9-900a-ddc0f2c27fab.png)

The landing page can be viewed on [github pages here](https://flynnduism.github.io/vscode-kubernetes-tools/) (my fork)

https://flynnduism.github.io/vscode-kubernetes-tools/

# follow up:

- [x] review the copy and layout
- [ ] get a better screenshot w/deeper cluster data
- [x] update the [baseURL](https://github.com/flynnduism/vscode-kubernetes-tools/blob/landing-page/site/config.toml#L1) to use the azure fork

itowlson commented on site/themes/vscode/README.md:
```diff
@@ -0,0 +1,12 @@
+![Theme Screenshot](https://raw.githubusercontent.com/deislabs/cnab.io/master/themes/cnab/img/screenshot.png?token=AAp4ctyXT6Xuaer1Vm8SXT0HscNhS5WGks5cBvT9wA%3D%3D)
+
+CNAB Theme
```
Should we use a different name here, or is this the name of the theme?  I'm guessing it's not very visible so won't cause confusion.

itowlson commented on site/themes/vscode/layouts/index.html:
```diff
@@ -0,0 +1,103 @@
+        </h1>
+        <!-- <h2>Access key parts of the Kubernetes ecosystem right inside your code editor.</h2> -->
+        
+        <p>The <a href="https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-kubernetes-tools">Kubernetes Tools</a> extension for VS Code bundles together a whole ecosystem of useful open source tools, making it easy for you to access and manage key parts of your app and microservices without leaving your code.</p>
```
I'm not sure we 'bundle a whole ecosystem' - longer term we will have packs that bundle larger parts of the ecosystem but the core is only Helm, and Draft.  @squillace may have thoughts on the wording here.

flynnduism commented:
Thanks @itowlson. Changes made:

* added `site` to `.vscodeignore`
* reworded that bit around _"ecosystem"_ to make it simpler. awaiting ralpha judgements
* fixed the theme description

itowlson commented:
@flynnduism @squillace Is this ready to merge, so we can have it live in time for the Kubecon talk?

flynnduism commented:
@itowlson happy to proceed if @squillace is satisfied

itowlson commented on site/themes/vscode/layouts/index.html:
```diff
@@ -0,0 +1,104 @@
+        <h2>Additional info:</h2>
+        <ul>
+          <li>
+            <a href="https://github.com/Azure/vscode-kubernetes-tools/tree/master/docs">Documentation</a>
```
Can we change this to https://github.com/Azure/vscode-kubernetes-tools/tree/master/docs/extending please?

itowlson commented on site/themes/vscode/layouts/index.html:
```diff
@@ -0,0 +1,104 @@
+          <li>
+            <a href="https://github.com/Azure/vscode-kubernetes-tools-api">NPM Package</a>
+          </li>
+        </ul>
```
Could we also add a link to the API samples please?  https://github.com/Azure/vscode-kubernetes-tools-api-samples

itowlson commented on site/themes/vscode/layouts/index.html:
```diff
@@ -0,0 +1,104 @@
+        <h1>Kubernetes Tools API</h1>
+        <h2>Custom tooling on top of the Kubernetes Tools extension.</h2>
+
+        <p>In building the kubernetes-tools extension, we've extracted a helpful <a href="https://github.com/Azure/vscode-kubernetes-tools-api">kubernetes-tools-api</a> layer and published it <a href="https://github.com/Azure/vscode-kubernetes-tools-api">here</a>, with the aim of enabling the development of more custom Kubernetes tools.</p>
```
The two links here are probably redundant - you could change the 'published it here' link to point to the NPM package?

flynnduism commented:
thanks for the comments @itowlson - I've made those url changes as suggested!

itowlson commented:
Thanks for the merge - sorry for dropping the ball on this.
